consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/api/IBreathingSpace.java

```diff
@@ -5,5 +5,7 @@
 
 public interface IBreathingSpace {
```

Comment:
To avoid confusion with the request type and make it more specific, rename this interface to ```IBreathingSpaceService```

Reply:
done